src/ConnectedKubernetes/test/AzConnectedKubernetes.Tests.ps1

```diff
@@ -18,7 +18,7 @@ Describe 'AzConnectedKubernetes' {
             $config.ProvisioningState | Should -Be 'Succeeded'
 
             # Clear helm azure-arc environment
-            helm delete azure-arc --no-hooks
+            helm delete azure-arc -n azure-arc-release --no-hooks
```

Comment:
Have you tested your command on you local? 
- To run command on your local, please use `./run-module.ps1` after `build-module.ps1`
- To record test cases, remove -Skip from It 'CreateExpanded' -skip and I'm expecting there is a json file under test folder.

Test always is the best way to ensure quality of commands. Your test case is always skipped with -Skip Tag

Reply:
Thanks for pointing out this. We've tested the code with integration test and end-to-end test with the product team for GA.
I think the -Skip was there because of the test case needs pre-requisites on the environment before it started.
[Prepare your machines for AKS Edge Essentials](https://learn.microsoft.com/en-us/azure/aks/hybrid/aks-edge-howto-setup-machine)
[Create a kubernetes cluster on the machine](https://learn.microsoft.com/en-us/azure/aks/hybrid/aks-edge-howto-single-node-deployment)

Reply:
Got it. I'd recommend preparing the environment in https://github.com/Azure/azure-powershell/blob/e9bdcf4f2844c8f0665ec230d5c44b0a1f44ed89/src/ConnectedKubernetes/test/utils.ps1#L16 if possible or make this test as -Live

Reply:
Since the dev and test env are separated at this moment. 
Maybe we can enhance the test once the product have better integration with the dev machine.